### DISCUSSION TITLE
How would I select a random user who reacted to a message?

### DISCUSSION THREAD

**Discussion by Nazuh2** [DSharpPlus/DSharpPlus#1117] [Q&A] [answered]
How would I select a random user who reacted to a message?
This is what I have now and its doesn't work
![image](https://user-images.githubusercontent.com/34795464/135761537-a742decd-13d6-4479-b937-72b01b18f65e.png)

**Comment by Naamloos** [comment, marked as answer]
Select isn't going to do anything for you, you'll want an RNG that has a range from 0 to the collection length.

**Reply by Nazuh2** [reply]
So how would I implement that?

**Reply by Naamloos** [reply]
> 
> 
> So how would I implement that?

that's up to you. These are basic things.

**Reply by Kaoticz** [reply]
Use `ElementAt`, then pass a number between 0 and the length of the collection.
(Also, for the love of God, don't use `.Result`)

**Reply by ProfDoof** [reply]
> So how would I implement that?

If you want help with questions like that join the C# Discord server https://discord.gg/csharp and ask there, join the D#+ server https://discord.gg/dsharpplus and ask in the programming channel or Google `how to select a random object from a list C#` (the third one should be your first step)

**Reply by Nazuh2** [reply]
> 
> 
> > So how would I implement that?
> 
> If you want help with questions like that join the C# Discord server https://discord.gg/csharp and ask there, join the D#+ server https://discord.gg/dsharpplus and ask in the programming channel or Google `how to select a random object from a list C#` (the third one should be your first step)

Oh I didn't realize there was a discord for this. I'll check that out thank you.
